### PR TITLE
Fix/law enforcement [Changed bal to b]

### PR DESCRIPTION
**Pull request by AmritKumar**
This PR is unrelated to QSP.

### CODE REVIEW

File: contracts/xsgd_contract.scilla [hunk 1]
```diff
@@ -372,7 +372,7 @@ transition lawEnforcementWipingBurn(address : ByStr20, initiator : ByStr20)
         currentSupply <- totalSupply;
         new_supply = builtin sub currentSupply bal;
```
Comment (junhaoyap) [inline contracts/xsgd_contract.scilla]: Should this also be changed to `b`? @AmritKumar

File: contracts/xsgd_contract.scilla [hunk 2]
```diff
@@ -44,7 +44,7 @@ contract XSGDContract
 field owner : ByStr20 = init_owner
 field pauser : ByStr20 = init_owner
 field masterMinter : ByStr20 = init_owner
-field paused : Bool = False
+field paused : Bool = True
```
Comment (junhaoyap) [inline contracts/xsgd_contract.scilla]:
```suggestion
field paused : Bool = False
```

File: contracts/xsgd_contract.scilla (hunk 1 above)
Comment (junhaoyap) [reply contracts/xsgd_contract.scilla]:
```suggestion
        new_supply = builtin sub currentSupply b;
```

**Comment by junhaoyap** [comment]
Thanks @AmritKumar

**Review by jjcnn — APPROVED** [review]

**Pull request merged**